README.rst

```diff
@@ -0,0 +1,223 @@
+* Mypy_: Static typing
+* Coverage_: Code coverage
+* Poetry_: Easy packaging and dependency management, set up to isolate the different
+  test dependencies (unit tests, bdd, ...) for speedy test runs
+* Pyenv_: Simple Python version management to install multiple versions on your host
+* Make_: Used as single doorway to all your tasks:
+
+  * For setup (development environment and machine, CI/CD pipeline job environments)
+  * For test tasks
+  * For the whole development and release lifecycle
+
+* Tox_ testing: Set up to easily test for Python 3.6, 3.7, 3.8 and used as single source
+  for all your test invocations
+* Sphinx_ docs: Documentation ready for generation with, for example, `Read the Docs`_
+* Gitlab-CI_: CI-CD pipeline with:
+
+  * Test phase running all tests in parallel jobs
+  * Release phase (see semantic-release below)
+
+* Semantic-release_: Pre-configured version bumping, changelog generation, auto-release
+  to PyPI_ and Gitlab (yes... not a Python written tool... but works *really* well and
+  has way more advanced features than any of the existing Python tools)
+* Command line interface (optional) using Cleo_ (default), Click_ or Argparse_
+
+
+Build Status
+-------------
+
+Linux:
+
+.. image:: https://gitlab.com/opinionated-digital-center/cookiecutter-pypackage/badges/master/pipeline.svg
+    :target: https://gitlab.com/opinionated-digital-center/cookiecutter-pypackage/pipelines
+    :alt: Linux build status on Gitlab CI
+
+Windows:
+
+This template is not tested on Windows yet, and some feature will most probably not be
+working on it, unless you use `Windows Subsystem for Linux`_.
+
+Quickstart
+----------
+
+Install the latest Cookiecutter if you haven't installed it yet (this requires
+Cookiecutter 1.4.0 or higher; see the `installation doc
+<https://cookiecutter.readthedocs.io/en/latest/installation.html>`_ for more details)::
+
+    # On macOS
+    $ brew install cookiecutter
+
+    # On linux and other systems
+    $ pip install -U cookiecutter
+
+Generate a Python package project::
+
+    $ cd your/projects/root/dir
+    $ cookiecutter https://github.com/opinionated-digital-center/cookiecutter-pypackage.git
+
+Then:
+
+* If you have not installed Pyenv_ and Poetry_ on your machine yet, you can use the
+  following ``make`` target to do so::
+
+    $ make setup-dev-host
+
+  Don't forget to restart your shell or source your shell configuration file.
+  For example for Bash::
+
+    $ source ~/.bashrc
+
+  You might want to tell Poetry to create your virtualenvs in your project directory
+  (makes it easier to add the virtualenv to your IDE)::
+
+    $ poetry config virtualenvs.in-project true
+
+* Set up you development environment::
+
+    # Minimal setup
+
+    $ make setup-dev-env-minimal
+
+    # OR alternatively, full setup (allows for IDE completion)
+
+    $ make setup-dev-env-full
+
+* Create a repo on Github or Gitlab (cloud or hosted) and put it there.
```

Comment:
"Put it there" is ambiguous : what ? and where ?

Reply:
Made changes, please review